recipes/fett/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Where did this license come from? I can't seem to find this in the upstream repository.

Reply:
Wasn't sure quite what to do about the license. The PyPi metadata (https://pypi.org/pypi/fett/json) says MIT, but there's no license file, so I faked it. Set the year to the first commit in the source repo and added the original author to it.

Not the right thing to do? Is it possible to just set `license: MIT` w/o a `license_file`? Honestly, didn't try that.

Reply:
The right first step is to open an issue on the upstream GitHub issue and ask for some clarification (i.e. says "MIT" but there's no license file). IANAL, but I'm really not sure if we can take a minor metadata declaration of "MIT" as a legal document allowing us to distribute code. And without that, in the US, the author(s) retain all rights.

Reply:
Yeah, makes sense. I've opened an issue w/ the upstream repo to add a license file. The last modification to the repo was a year ago, but maybe I'll get lucky and the owner will respond to my issue.